lib/logstash/filters/prune.rb

```diff
@@ -16,11 +16,11 @@ class LogStash::Filters::Prune < LogStash::Filters::Base
   config_name "prune"
 
   # Trigger whether configuration fields and values should be interpolated for
-  # dynamic values.
+  # dynamic values (when resolving %{some_field}).
   # Probably adds some performance overhead. Defaults to false.
   config :interpolate, :validate => :boolean, :default => false
 
-  # Include only fields only if their names match specified regexps, default to empty list which means include everything.
+  # Include only fields and tags only if their names match specified regexps, default to empty list which means include everything.
```

Comment:
The way you've reworded this sentence (and the corresponding one on line 32) could give the impression that the pruning acts on tag values themselves rather than the `tag` field, i.e. that you could say `whitelist_names => ["foo"]` to keep the "foo" tag but remove all other tags.

Would it be a good idea to reword this to avoid that ambiguity? Maybe add a sentence (here or elsewhere) similar to "The `tags` field that stores the event's tags is also subject to pruning."